MemberExpectations.md

```diff
@@ -0,0 +1,58 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the code of conduct but that supports
+the vision, mission and operating principles of the Node.js project.
+
+While members have the right and responsibility to exercise independent
+judgment and to express dissenting opinions during discussions within
```

Comment:
judgement

Reply:
_judgement_ is the British spelling, but Americans prefer _judgment_. In the main repo, the [style guide](https://github.com/nodejs/node/blob/5f223759229a112af41508c470d683152523c6e3/doc/STYLE_GUIDE.md) indicates that American spellings are preferred, so if that applies here, _judgment_ is the better choice. There's also a _behaviour_ later in the doc that should be changed to _behavior_ if the American English thing applies to all our docs.

Reply:
agreed, spelling was never my strong point and I often mix Canadian and US spelling, will fix up.